bin/process_partis.py

```diff
@@ -246,13 +243,12 @@ def processed_data(args):
     """Uses args to find the correct partition, cluster pair and all associated information. Cluster
     information is returned as by process_cluster."""
 
-    #print("calling utils.read_output with args:", args.partition_file, args.glfo)
-    file_glfo, annotation_list, cpath = utils.read_output(args.partition_file, glfo=args.glfo)
+    glfo = None if utils.getsuffix(args.partition_file) == '.yaml' else glutils.read_glfo(args.glfo_dir if args.glfo_dir else default_glfo_dir, args.locus)
```

Comment:
@psathyrella worth breaking the `else` into blocks here for a `warn` about using `default_glfo_dir` echoing your comment above if it really is a last resort to use that glfo?

Reply:
nah, the worst case is really just that there'll be a novel allele in the proper germline set that'll be missing form the default one, which'll cause a prompt crash with an informative error message.